.gitignore

```diff
@@ -140,30 +134,40 @@ DartConfiguration.tcl
 LibeventConfig.cmake
 LibeventConfigVersion.cmake
 LibeventTargets*.cmake
+Uninstall.cmake
 bin/
 cmake_install.cmake
-Uninstall.cmake
 compile_commands.json
+event.dir
+event_core.dir
+event_extra.dir
+install_manifest.txt
 lib/
 tmp/
 verify_tests.sh
 verify_tests.bat
-event.dir
-event_core.dir
-event_extra.dir
 *.vcxproj
 *.sln
 *.filters
-install_manifest.txt
-test-export/build
 
-# ninja
+# files generated by Xcode (`cmake -B build -G Xcode && cmake --build build`)
+*.resp
+*.d
+*.dia
+Debug/
+Release/
+XCBuildData/
+CMakeScripts/
+libevent.xcodeproj/
+
+# files generated by Ninja (`cmake -B build -G Ninja && cmake --build build`)
 build.ninja
 rules.ninja
 .ninja_deps
 .ninja_log
 
-# make dist
+# make dist (`mkdir dist; cd dist && ../configure && make distcheck`)
+/dist/
```

Comment:
This is not required, everything inside it **should be** ignored anyway.

Reply:
2ea33f478edc1c7b9b0f3d1210a9bbffa02ca693